### PR TITLE
Fix updates needed for new project

### PR DESCRIPTION
**Pull request by BillyG83**
Tag custom padding added

### CODE REVIEW

File: packages/core/src/lib/Tag/Tag.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ const Tag: React.FC<TagProps> = ({
     hasCancel = false,
     id,
     onCancelClick,
+    padding,
     text = 'Tag',
     theme = 'regular',
     tone = 'light',
```
[the remark this answers is not in the document]
Comment (AbhinavMV) [reply packages/core/src/lib/Tag/Tag.tsx:17]: Also in this file below on line 48, old color is being used. Looks like we missed this file.
`<Cross
           title="cross icon"
           titleId="tag cross icon"
           fill={colors.grey}
 />`

[the remark this answers is not in the document]
Comment (BillyG83) [reply packages/core/src/lib/Tag/Tag.tsx:17]: I THOUGHT YOU WERE SUPPOSED TO BE TAKING THE WEEKEND OFF!!!

LOL! great catches TY, it was late 😅

[the remark this answers is not in the document]
Comment (BillyG83) [reply packages/core/src/lib/Tag/Tag.tsx:17]: fixed TY @AbhinavMV 🙇

File: packages/core/src/lib/InputNew/Input.styles.ts [hunk 2]
```diff
@@ -100,6 +109,10 @@ const DivStyled = styled.div<TInputProps>`
 
     &:hover {
         border-color: ${color.navy30};
+        ${(p) =>
+            p.customize?.onHover && p.customize?.onHover === 'lighten'
+                ? 'filter: brightness(1.1)'
+                : 'filter: brightness(10%)'};
```
[the remark this answers is not in the document]
Comment (BillyG83) [reply packages/core/src/lib/InputNew/Input.styles.ts]: fix Ty bud

**Pull request merged**